app/controllers/posts_controller.rb

```diff
@@ -8,4 +8,19 @@ def show
     @post = Post.find(params[:id])
     @comments = @post.comments
   end
+
+  def new
+    @post = Post.new
+    # @user=current_user
```

Comment:
- [ ] Please kindly remove commented code from your codebase

Reply:
Hi @AnselemOdims 

Thank you for noticing, I forgot about that